src/utils/Fetcher.ts

```diff
@@ -24,7 +23,19 @@ export class Fetcher {
     this.customDefaults = mergeRequestOptions(this.customDefaults, overrideDefaults)
   }
 
-  fetchJson(url: string, options?: RequestOptions): Promise<any> {
+  fetch(url: string, options: Partial<CompleteRequestOptions>): Promise<Response> {
+    return fetchInternal(url, {
+      method: 'get',
+      // it is better to not assume how this generic fetch will be used, not sending retries
+      attempts: 1,
+      timeout: '100d',
+      waitTime: '0s',
+      ...mergeRequestOptions(this.customDefaults, options),
+      ...options
```

Comment:
do you need this, if you are already merging in the line before?

Reply:
Gud catch